webpack/assets/javascripts/react_app/redux/actions/common/forms.js

```diff
@@ -0,0 +1,78 @@
+import { SubmissionError } from 'redux-form';
+import fetch from 'isomorphic-fetch';
+import { addToast } from '../toasts';
+import $ from 'jquery';
+
+const fieldErrors = ({ error }) => {
+  let errors = error.errors;
+
+  if (errors.base) {
+    errors._error = errors.base;
+    delete errors.base;
+  }
+  return new SubmissionError(errors);
+};
+
+const checkErrors = response => {
+  if (response.ok) {
+    return response;
+  }
+  if (response.status === 422) {
+    // Handle invalid form data
+    return response.json().then(body => {
+      throw fieldErrors(body);
+    });
+  }
+  throw new SubmissionError({
+    _error: [`${__('Error submitting data:')} ${response.status} ${__(response.statusText)}`]
+  });
+};
+
+const getcsrfToken = () => $('meta[name=csrf-token]').attr('content');
```

Comment:
I don't think, this is the proper way to do this. Isn't there a way to do this without jQuery? How about passing this similar to the provider in the `MountingService`?

Reply:
without jquery:
```js
document.querySelector('meta[name="csrf-token"]').content
```
i think passing it to the mount service is a lot of boilerplate code? TBH I'm not sure if its needed to begin with? i didnt re-read the auth code, but it might not even be need in some form?

Reply:
also, passing it as a prop in mounting sevice feels like its boilerplate code, especially since its already on the dom.